src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.AssemblyResolution.targets

```diff
@@ -257,11 +257,25 @@ _ResolveAssemblies MSBuild target.
       <_ExcludedNativeLibraries Condition=" '$(_AndroidRuntime)' == 'CoreCLR' " Include="libmono-android.release" />
     </ItemGroup>
 
+    <!-- Let's keep the item group close to the task that uses them, the list is irrelevant for
+         anything else. The group MUST contain only shared libraries from the runtime pack.
+         DO NOT modify the group unless you're absolutely sure what you're doing -->
+    <ItemGroup Condition=" '$(_AndroidRuntime)' == 'CoreCLR' " >
+      <_KnownRuntimeNativeLibrary Include="libSystem.Globalization.Native.so" />
+        <_KnownRuntimeNativeLibrary Include="libSystem.IO.Compression.Native.so" />
+        <_KnownRuntimeNativeLibrary Include="libSystem.Native.so" />
+        <_KnownRuntimeNativeLibrary Include="libSystem.Security.Cryptography.Native.Android.so" />
+        <_KnownRuntimeNativeLibrary Include="libclrjit.so" />
+        <_KnownRuntimeNativeLibrary Include="libcorclr.so" />
+    </ItemGroup>
```

Comment:
Are there additional `.so` files in the runtime pack? Should we let the runtime control this list somehow?

I'm wondering how it will break when they introduce a new one.

Reply:
There are additional files in the runtime pack, yes, but we don't need them. I proposed that the pack contains a manifest, but there have been just preliminary discussions so far. Runtime pack having the manifest is the best solution, no doubt.